doc/tutorials/space_shooter/app/pubspec.yaml

```diff
@@ -6,20 +6,15 @@ publish_to: 'none'
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
+  sdk: '>=2.18.2 <3.0.0'
```

Comment:
Seems like we're requiring `2.17.0` min version for other packages, so perhaps let's keep this consistent?